src/account/postAccount.test.ts

```diff
@@ -0,0 +1,17 @@
+import * as postAccount from './postAccount';
+
+describe('Add favourite movie to the account', () => {
+
+    test('Verify that the status code of the response is 201', async () => {
+        const response = await postAccount.addAccountFavourite(390);
```

Comment:
You need to check if the movie or smth with ID 390 was really added.

Reply:
Fixed